edi_account/models/account_move.py

```diff
@@ -8,12 +8,13 @@ class AccountMove(models.Model):
     _name = "account.move"
     _inherit = ["account.move", "edi.exchange.consumer.mixin"]
 
-    disable_edi_auto = fields.Boolean(
-        help="When marked, EDI could be avoided",
-        readonly=True,
+    edi_auto_disabled = fields.Boolean(
         states={"draft": [("readonly", False)]},
     )
 
+    disable_edi_auto = fields.Boolean(related="edi_auto_disabled")
+    # TODO: remove this field at version 16.0
```

Comment:
why not 14.0?

Reply:
Because edi_account is already migrated to 14.0 and 15.0. I was leaving it just for security reasons, maybe someone has already custom modules based on this module at those versions. Do you think it is better to remove it at 14.0?

Reply:
IMO: yes. For existing installations you probably need a small migration script. Here you can define a post install hook that can be used later for upgrades too (you can import an hook from the `migrations` folder but you cannot import viceversa).